package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hof",
   "description": "A bootstrap for HOF projects",
-  "version": "20.4.2",
+  "version": "20.4.3",
```

Comment:
Are you looking to publish a test version i.e. beta?you wouldn't want to publish this for the proper version of HOF as it'll impact everyone.